consuming-webservices/Networking/RequestHandler.swift

```diff
@@ -45,12 +45,44 @@ struct HTTPRequestHandler: RequestHandler {
     var body: Any?
     
     func execute( callback: @escaping (Result<Any>) -> Void) {
-        var returnValue = 0
-        if let body = body as? [String: String] {
-            returnValue = body["args"] == "on" ? 1 : 0
+        guard let url = URL(string: path) else { return }
```

Comment:
Hey Stephen, I'm not familiar with this pattern you are referring to. Could you show me what that would look like?